sdk/src/main/java/com/talkable/sdk/TalkableOfferFragment.java

```diff
@@ -323,12 +328,28 @@ public void shareOfferViaFacebookMessage(String claimUrl) {
         FacebookUtils.shareViaMessenger(this, claimUrl);
     }
 
+    public void shareOfferViaWhatsapp(String message) {
```

Comment:
Naming should be consistent between this and `isWhatsAppAvailable`.

```suggestion
    public void shareOfferViaWhatsApp(String message) {
```